src/routes/tickets/controllers/replyTicket.ts

```diff
@@ -1,65 +1,66 @@
 import { getDB } from '@database/db';
 import { validationResult, matchedData } from 'express-validator';
 import { ObjectId } from 'mongodb';
-import { populatePersonnelView } from '../util/populatePersonnelView';
-import { ITicket, TicketStatus } from './ticketController';
 import { v4 as uuid } from 'uuid';
 import { responseGenerator } from '@util/responseGenerator';
+import { populatePersonnelView } from '../util/populatePersonnelView';
+import { TicketStatus } from './ticketController';
+import { Request, Response } from 'express';
 
-const replyTicket = async (req, res): Promise<ITicket> => {
-	const errors = validationResult(req);
-	const data = matchedData(req);
+const replyTicket = async (req: Request, res: Response) => {
+  const errors = validationResult(req);
+  const data = matchedData(req);
 
-	if (!errors.isEmpty()) {
-		return res.status(400).send({ success: false, msg: 'Bad request', errors: errors.array() });
-	}
+  if (!errors.isEmpty()) {
+    return res.status(400).send({ success: false, msg: 'Bad request', errors: errors.array() });
+  }
 
-	const _id = new ObjectId(data.ticketId);
-	const now = new Date();
-	const replyId = uuid();
+  const _id = new ObjectId(data.ticketId);
+  const now = new Date();
+  const replyId = uuid();
 
-	const updatedData = await getDB().tickets.findOneAndUpdate(
-		{ _id },
-		{
-			$set: {
-				status: TicketStatus.updated,
-			},
-			$push: {
-				messages: {
-					id: replyId,
-					authorId: req.user.sub,
-					author: req.user.name,
-					text: data.text,
-					createdAt: now,
-				},
-				personnelView: {
-					id: replyId,
-				},
-			},
-		},
-		{
-			returnOriginal: false,
-		},
-	);
+  const updatedData = await getDB().tickets.findOneAndUpdate(
+    { _id },
+    {
+      $set: {
+        status: TicketStatus.open,
```

Comment:
Out of curiosity, why the change from updated to open?

Reply:
Forgot to revoke this change, will do so now.